src/main/Member.js

```diff
@@ -243,18 +266,22 @@ export default class Member {
      * oauthBankAuthorization
      * @return {Promise} accounts - Promise resolving the the Accounts linked
      */
-    linkAccounts(authorization) {
+    linkAccounts(
+        authorization: OauthBankAuthorization | any
```

Comment:
`| string`?

Reply:
tried, but flow doesnt seem to be smart enough to differentiate between the two types through the if statement below, so it throws that string doesn't have toJSON. I think this is fine because the old flow will be deprecated soon anyway right? We will just remove it